aspnetcore/fundamentals/websockets.md

```diff
@@ -37,35 +39,16 @@ This article explains how to get started with WebSockets in ASP.NET Core. [WebSo
 
 * For supported browsers, see https://caniuse.com/#feat=websockets.
 
-## When to use WebSockets
-
-Use WebSockets to work directly with a socket connection. For example, use WebSockets for the best possible performance with a real-time game.
-
-[ASP.NET Core SignalR](xref:signalr/introduction) is a library that simplifies adding real-time web functionality to apps. It uses WebSockets whenever possible.
-
-## How to use WebSockets
+## NuGet package
 
-* Install the [Microsoft.AspNetCore.WebSockets](https://www.nuget.org/packages/Microsoft.AspNetCore.WebSockets/) package.
-* Configure the middleware.
-* Accept WebSocket requests.
-* Send and receive messages.
+Install the [Microsoft.AspNetCore.WebSockets](https://www.nuget.org/packages/Microsoft.AspNetCore.WebSockets/) package.
```

Comment:
WebSockets is in the shared framework, so it's automatically available in all ASP.NET Core 2.1+ apps. I'd remove this section entirely. Or at least make it 2.0 or lower.